src/docs/PML/PMLUserGuide.adoc

```diff
@@ -4714,9 +4715,24 @@ image::Fetch_Satellite_Images.png[align=center]
 | `SEARCH_ENGINE`
 | Defines an engine to search and download satellite images.
 | List [Peps, Copernicus, All] (default=Copernicus)
+| `IMPORT_FROM`
+| Downloads files or folders to the data space.
+| String (default=PA:GLOBAL_FILE)
+| `TIME_TO_RETRIEVE_IN_SECONDS`
+| Defines time in second to request an offline product from the Long Term Archive (LTA) in the Copernicus Open Access Hub’s.
+| Int (default=900 seconds)
+| `TIME_TO_CHECK_ONLINE_IN_SECONDS`
+| Defines time in second to check if a product is online.
+| Int (default=1800 seconds)
+| `WALLTIME`
+|  Determines the maximum execution time of a task
```

Comment:
extra space before Determines the maximum and a . is missed by the end of the sentence